### PR TITLE
add target for script

### PR DESCRIPTION
**Pull request by brunoerg**

### CODE REVIEW

File: rust_bitcoin_lib/src/lib.rs [hunk 1]
```diff
@@ -100,6 +101,23 @@ pub unsafe extern "C" fn rust_bitcoin_psbt(data: *const u8, len: usize) -> *mut
...
+#[no_mangle]
+pub unsafe extern "C" fn rust_bitcoin_script(data: *const u8, len: usize) -> bool {
+    // Safety: Ensure that the data pointer is valid for the given length
+    let data_slice = slice::from_raw_parts(data, len);
+
+    let script: Result<(bitcoin::script::ScriptBuf, usize), encode::Error> = encode::deserialize_partial(data_slice);
+    match script {
+        Err(_) => false,
+        Ok(s) => {
+           if s.0.is_op_return() || s.0.len() > 10_000 {
```
Comment (hax0kartik) [inline rust_bitcoin_lib/src/lib.rs:113]: I think there are unintentional *minor* formatting issues in this line

File: targets/script.cpp [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#include <fuzzer/FuzzedDataProvider.h>
+#include <string>
```
Comment (hax0kartik) [inline targets/script.cpp:2]: This include is not required

File: targets/script.cpp [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+#include <iostream>
```
Comment (hax0kartik) [inline targets/script.cpp:3]: +1 not required

File: rust_bitcoin_lib/src/lib.rs (hunk 1 above)
Comment (brunoerg) [reply rust_bitcoin_lib/src/lib.rs:113]: which one?

File: targets/script.cpp (hunk 2 above)
Comment (brunoerg) [reply targets/script.cpp:2]: thanks

File: rust_bitcoin_lib/src/lib.rs (hunk 1 above)
Comment (brunoerg) [reply rust_bitcoin_lib/src/lib.rs]: got it, fixed

**Pull request merged**